commit e6911affa416dc4e0c0b3f04cbe6b02ce13277f1
Author: Xu Jia <xujia39@huawei.com>
Date:   Wed Dec 22 17:06:58 2021 +0800

    xfrm: Add support for SM3 secure hash
    
    This patch allows IPsec to use SM3 HMAC authentication algorithm.
    
    Signed-off-by: Xu Jia <xujia39@huawei.com>
    Signed-off-by: Steffen Klassert <steffen.klassert@secunet.com>

diff --git a/include/uapi/linux/pfkeyv2.h b/include/uapi/linux/pfkeyv2.h
index d65b11785260..798ba9ffd48c 100644
--- a/include/uapi/linux/pfkeyv2.h
+++ b/include/uapi/linux/pfkeyv2.h
@@ -309,6 +309,7 @@ struct sadb_x_filter {
 #define SADB_X_AALG_SHA2_512HMAC	7
 #define SADB_X_AALG_RIPEMD160HMAC	8
 #define SADB_X_AALG_AES_XCBC_MAC	9
+#define SADB_X_AALG_SM3_256HMAC		10
 #define SADB_X_AALG_NULL		251	/* kame */
 #define SADB_AALG_MAX			251